lib/ansible/modules/network/junos/junos_package.py

```diff
@@ -81,6 +81,7 @@
   - This module requires the netconf system service be enabled on
     the remote device being managed.
   - Tested against vSRX JUNOS version 15.1X49-D15.4, vqfx-10000 JUNOS Version 15.1X53-D60.4.
+  - Works with local connection type only.
```

Comment:
What's the plan for this, since it's connection local only

Reply:
Since these modules will use `junos-eznc` library and not the persistent connection framework it will continue to be supported with connection=local
In future, we might want to move these modules to a role within ansible-network.